README.md

```diff
@@ -32,6 +32,12 @@ Run tests
 ```shell
 $ make test
 ```
+  
+To test the Nexus Pro features you will need to install a Nexus Pro license, restart nexus and then enable the additional tests:
+
+```shell
+$ export SKIP_PRO_TESTS=false
```

Comment:
please set this as default behavior. Our test-pipeline use a nexus pro instance now.